src/node.h

```diff
@@ -328,12 +328,14 @@ class NODE_EXTERN MultiIsolatePlatform : public v8::Platform {
 
 enum IsolateSettingsFlags {
   MESSAGE_LISTENER_WITH_ERROR_LEVEL = 1 << 0,
-  DETAILED_SOURCE_POSITIONS_FOR_PROFILING = 1 << 1
+  DETAILED_SOURCE_POSITIONS_FOR_PROFILING = 1 << 1,
+  SHOULD_NOT_SET_PROMISE_REJECTION_CALLBACK = 1 << 2
 };
 
 struct IsolateSettings {
   uint64_t flags = MESSAGE_LISTENER_WITH_ERROR_LEVEL |
-      DETAILED_SOURCE_POSITIONS_FOR_PROFILING;
+      DETAILED_SOURCE_POSITIONS_FOR_PROFILING |
+      SHOULD_NOT_SET_PROMISE_REJECTION_CALLBACK;
```

Comment:
is this supposed to be set by default?

Reply:
It’s not supposed to be set by default, thanks for noticing that :+1:

Reply:
whoops, ty for the catch!